### DISCUSSION TITLE
Questão TI21#2

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#8] [Q&A] [answered]
Como você pode centralizar um elemento verticalmente em CSS?

**Comment by Arrowaaa** [comment]
Para centralizar um elemento verticalmente em CSS : 
Você pode usar a propriedade position(absolute) e transform(translateY(-50%))
ou 
Usando display:flex ;
flex-direction: colunm;
e justify-content: center;

**Reply by Guilhlm** [reply, marked as answer]
voce pode usar align itens: center em css

**Comment by RenonHG** [comment]
Utilizando o comando align-items: center;

**Comment by Malvadaokkj** [comment]
Você pode centralizar utilizando o que irei citar abaixo
position: absolute que pode centralizar um elemento verticalmente
line-height serve para centralizar o texto verticalmente
Entre outros

**Comment by riichardgabriel** [comment]
Posicionamento Vertical

Este método utiliza a propriedade display: table na DIV externa e as propriedades display:table-cell e vertical-align: middle na div interna para que os elementos se comportem como elementos de tabela, permitindo que o conteúdo seja centralizado normalmente.
